unlock-app/src/components/pages/Home.js

```diff
@@ -13,10 +15,12 @@ export default class Home extends PureComponent {
           Unlock is a protocol which enables creators to monetize their content with a few lines of code in a fully decentralized way.
         </Headline>
 
-        <Action>
-          <HomepageButton>Go to Your Dashboard</HomepageButton>
-          <ButtonLabel>Requires a browser with an Ethereum wallet</ButtonLabel>
-        </Action>
+        <Link to={'/dashboard'}>
```

Comment:
You'll know what I say about nesting components!... but I guess you tried without?

Reply:
I did, yeah. I have some feature requests for the next version of styled-components.

Reply:
lol. happy to hear them!